sunpy/map/mapbase.py

```diff
@@ -201,7 +201,7 @@ def __init_subclass__(cls, **kwargs):
         if hasattr(cls, 'is_datasource_for'):
             cls._registry[cls] = cls.is_datasource_for
 
-    def __init__(self, data, header, plot_settings=None, **kwargs):
+    def __init__(self, data, header, plot_settings=None, mask=None, **kwargs):
```

Comment:
What happens to kwargs now?

Reply:
slipped right through my mind, should be fixed now👍